src/services/shims.ts

```diff
@@ -322,8 +323,19 @@ module ts {
         }
     }
 
+    function serializeResultAsJson(target: Shim, key: string | symbol, descriptor: PropertyDescriptor) {
+        let action = descriptor.value;
+        descriptor.value = function () {
+            let _arguments = arguments;

```

Comment:
for _arguments, i did not want to add a loop and an array allocation for every LS call.